Automattic/wp-calypso
# Tailored Flows: Progress bar based on user logged in

#### Proposed Changes

 Context: pdtkmj-pL-p2#:~:text=It%E2%80%99s%20strange%20that%20the%20progress%20bar%20starts%20almost%20a%20third%20filled%20already%20on%20this%20screen.%20Might%20be%20ok%20removing%20it%20here

In the tailored flows, if the user goes through the flow logged in, he directly jump from `intro` step (where progress is zero) to `newsletter`/`linkInBio setup`  step, step number  2. This causes the progress bar to jump ahead a lot.

This PR solves that, removing the user step from the progress bar calculations if the user is logged in.

Before:
![image](https://user-images.githubusercontent.com/52076348/190384069-d22a95dc-00a5-4083-92b1-1c9fc21b22c2.png)

After:
![image](https://user-images.githubusercontent.com/52076348/190384068-6c0ded3e-23fc-4a19-8f08-9d97da668f3a.png)


#### Testing Instructions



* Checkout branch and `yarn start`
* Test Newsletter Flow and Link in Bio flow, while Logged in. Progress bar should appear like in the screenshots
* Test both the flows in incognito, not logged in.

Related to #67369
Fixes #67369
